service/rest/src/main/java/nl/tudelft/wdm/group1/rest/web/OrderController.java

```diff
@@ -47,7 +47,7 @@
     }
 
     @PostMapping("/{id}/checkout")
-    public CompletableFuture<Order> checkoutOrder(@PathVariable(value = "id") UUID id) {
-        return kafka.performAction(new OrderCheckoutPayload(id));
+    public CompletableFuture<Order> readyOrder(@PathVariable(value = "id") UUID id) {
```

Comment:
Why should `/order/id/checkout` map to ready?

Reply:
![image](https://user-images.githubusercontent.com/23526707/58754022-caaa6100-84c8-11e9-9b53-74b8df61b516.png)
Because we agreed to do so (see image)
Checkout is after stock is checked and payment can be deducted

Reply:
Did you also replace the ORDER_CHECKED_OUT with ORDER_READY in stock consumer? I can fix it in my branch if you didn't.